api/services/BuildResolver.js

```diff
@@ -0,0 +1,73 @@
+const { Build, Site } = require('../models');
+const buildErrors = require('../responses/buildErrors');
+const GitHub = require('../services/GitHub');
+
+const rejectBuild = msg =>
+  Promise.reject({
+    status: 404,
+    message: msg,
+  });
+
+const getBranch = ({ user, site, branch }) =>
```

Comment:
maybe rename to `getBranchFromGitHub` for a little extra clarity, especially where this is used down below